src/quo/components/wallet/network_link/view.cljs

```diff
@@ -1,82 +1,151 @@
 (ns quo.components.wallet.network-link.view
   (:require
+    [oops.core :refer [oget]]
```

Comment:
can we move away from oops. it crashes too often 🙃 
cc @ulisesmac @seanstrom

Reply:
We can, of course. Any suggested approach?

Reply:
Maybe we can use `goog.object/get`?
I think that function will not throw, and we can always wrap/export it from our utils namespace if we want to replace it with something else later. Thoughts?

For context:
https://google.github.io/closure-library/api/goog.object.html
https://clojureverse.org/t/cljs-hidden-google-closure-library-gems/2321/2